File: 7KYU/Histogram - H1/solution.cs
using System.Linq;
public class Dinglemouse 
{
// results to dictionary keyed on n, Count(n)
  public static string Histogram(int[] results)
  {
    return string.Join("\n", results.Reverse()
                                    .Select((r, i) => $"{6-i}|{string.Concat(Enumerable.Repeat("#", r))}{GetEndCount(r)}")) + "\n";
  }
  
  public static string GetEndCount(int r)
  {
    if(r == 0)
      return "";
    else
      return $" {r}";
  }
}

File: 7KYU/Histogram - H1/solution.py
def getEndCount(n):
    if n == 0:
      return ""
    return " " + str(n);

def histogram(n):
    solution = []
    for i in range(1, 7):
      solution.append(str(7-i) + "|" + ("#" * n[6-i]) + getEndCount(n[6-i]))
    return "\n".join(solution)  + "\n"